pkg/server/server.go

```diff
@@ -0,0 +1,53 @@
+package server
+
+import (
+	"fmt"
+	"mseScraping/pkg/conf"
+	"mseScraping/utils"
+	"net/http"
+	"os"
+
+	"github.com/go-chi/chi/v5"
+	"github.com/joho/godotenv"
+)
+
+type HandleWithBody func(body any)
+
+type Server struct {
+	Conf   conf.Conf
+	Router *chi.Mux
+}
+
+func Build() Server {
```

Comment:
Can we parse vars from os env?

Reply:
Yes we can parse os env since we are getting it from `os.Getenv`